packages/extension-api/src/extension-api.d.ts

```diff
@@ -1042,6 +1042,18 @@ declare module '@podman-desktop/api' {
      * Whether or not to emit an OS notification noise when showing the notification.
      */
     silent?: boolean;
+    /**
+     * The type of the notification. Default value: info
+     */
+    type?: NotificationType;
+    /**
+     * displayed below the description, centered in the notification card. It contains actions (like markdown commands/buttons and links)
+     */
+    actions?: string;
```

Comment:
I think in the API we should not say where it'll be or if it'll be centered, etc as layout may change later

it's not clear to me what's the content of the actions field. Is it markdown content or name of a command that I want to invoke ?
Probably need to use `markdown` prefix as in other place to make sure we can provide markdown content there and maybe rename it to `markdownSomething` (like `markdownDetail`) as actions might look too specific on the purpose, you might just want to display something without any action

Reply:
Idk honestly. Based on Mo's mockup a notification could have a button associated to it, ( look at the third item https://github.com/containers/podman-desktop/issues/1230 )
But i think that it could be a link, or a couple of buttons. This is why i called them actions.